packages/shared-react/utils/assetUtils.ts

```diff
@@ -1,3 +1,3 @@
-export function getAssetUrl(assetId: string) {
-  return `/api/assets/${assetId}`;
+export function getAssetUrl(assetId: string, preview?: boolean) {
+  return `/api/assets/${assetId}${preview ? "?preview" : ""}`;
```

Comment:
as for why i am using the querystring "?preview" instead of "?preview=true" (or something similar):
The <Image> element would then URL encode the "=" sign, which breaks the URL for some reason and then route.ts will not be called anymore.